sdk/servicebus/service-bus/test/internal/atomXml.spec.ts

```diff
@@ -72,1065 +72,1067 @@ const mockServiceBusAtomManagementClient: ServiceBusManagementClient = new Servi
   "Endpoint=sb://test/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=test"
 );
 
-describe("atomSerializationPolicy", function() {
-  it("should throw an error if receiving a non-XML response body", async function() {
-    const request = new WebResource();
-    mockServiceBusAtomManagementClient.sendRequest = async () => {
-      return {
-        request: request,
-        status: 200,
-        headers: new HttpHeaders({}),
-        bodyAsText: `{ hello: "this is a JSON response body" }`
+describe("atom", () => {
```

Comment:
Maybe a more descriptive name? `ATOM Serializers` ?

Reply:
I can't disagree with the ATOM manager. Will change ;)

Reply:
Fixed.